consent-ui/src/app/errorsHandler/global-errors-handler.ts

```diff
@@ -24,7 +24,5 @@ export class GlobalErrorHandler implements ErrorHandler {
         severity: 'error'
       });
     });
-
```

Comment:
I don't see where the error is logged to console

Reply:
fixed